components/x-privacy-manager/.npmignore

```diff
@@ -0,0 +1,3 @@
+src/
+stories/
+rollup.js
```

Comment:
So if this file excludes the mentioned directories from getting included in the package when it is published, presumably the code that does get published is that in the `dist` directory?

Reply:
Spot on.
The compiled output is referenced explicitly by the following keys in `package.json`:
- `main`
- `module`
- `style`